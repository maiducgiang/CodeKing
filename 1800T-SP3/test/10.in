10
60
837 -711
156 573
470 -606
986 -313
686 -943
378 66
469 999
460 512
396 -529
908 -330
704 -155
421 676
419 -803
360 528
410 855
964 -831
660 914
404 -167
98 486
476 420
312 898
909 133
942 -481
587 -267
379 203
424 418
675 -625
568 -42
448 714
693 -981
741 -314
692 -68
92 -81
43 229
314 -660
574 343
287 970
37 -774
574 818
30 -404
466 -305
512 836
131 -285
806 667
424 -194
663 201
312 -18
592 791
963 296
993 -39
107 -558
474 -879
684 150
370 972
95 -683
121 -895
632 87
846 842
572 990
350 518
53
932 -392
156 375
481 138
324 354
276 883
327 199
834 -439
940 -291
301 567
632 -535
27 -58
896 -745
31 134
7 -542
172 554
766 -124
401 341
443 722
730 -879
419 566
653 788
694 -334
533 156
845 -858
403 281
467 301
303 -859
1 -648
568 496
189 -184
867 -293
190 304
439 -521
247 792
727 -228
917 166
58 141
586 -696
261 558
106 -332
849 202
718 971
808 889
130 959
465 -841
585 592
685 -68
210 826
21 167
110 962
727 49
826 -94
232 484
79
896 -131
441 -280
773 -365
184 -255
919 101
79 380
613 -69
986 566
887 -519
297 521
756 714
487 -372
38 922
386 -524
963 933
702 -1
584 -669
632 157
478 -406
244 -961
505 -349
385 120
693 801
213 678
724 -609
617 377
271 178
565 509
707 579
680 -481
858 -990
721 -272
272 -109
499 -728
135 6
95 427
611 -995
65 -114
385 707
502 210
259 370
741 -928
358 -814
424 800
230 605
101 -852
939 157
134 335
10 -634
992 -247
604 346
208 -295
716 -250
642 577
317 917
558 -808
429 -860
935 -50
982 -242
925 -369
640 545
646 675
410 -649
734 882
673 -397
302 457
930 728
223 668
851 841
480 -975
990 283
938 -194
948 -431
118 197
210 -400
696 515
585 135
490 -221
55 209
22
18 846
482 446
104 -262
636 688
955 -967
331 980
64 -775
716 305
650 -644
996 861
691 652
947 244
151 -14
777 93
418 21
210 989
658 8
124 222
468 -428
248 -354
540 466
686 -428
97
1 -1
583 253
649 -510
226 830
692 159
741 92
602 822
779 262
468 -339
713 953
842 -210
178 247
785 933
558 -958
632 -31
321 -392
43 -91
254 527
792 -771
56 60
498 962
83 857
46 313
324 737
311 -527
905 239
322 -277
370 954
846 11
908 -396
636 306
249 -72
334 773
388 -12
532 865
447 -486
95 -433
728 654
709 -514
77 -33
303 36
675 725
836 -764
471 -909
361 -639
733 750
89 668
991 -835
723 -348
4 -380
676 692
581 -792
843 -257
221 997
104 -482
387 -863
625 323
371 -454
629 -939
299 -595
382 -251
520 726
145 743
110 -497
580 392
507 15
492 521
471 728
814 436
712 -866
439 -792
15 593
872 -212
56 -943
952 -774
218 593
711 -445
459 486
499 -109
159 791
244 560
185 -628
361 564
513 -105
221 -740
183 854
661 273
442 955
430 715
737 -651
322 441
420 558
106 797
409 963
871 -252
789 -603
777 -568
84
615 45
13 408
854 818
364 213
679 -492
192 -689
704 996
321 141
680 -702
532 359
366 535
912 -627
376 173
179 -955
93 285
16 345
413 239
232 -214
593 -365
988 -129
764 709
922 305
461 692
554 -635
908 -178
220 -790
746 606
996 -562
740 135
167 -425
427 848
559 -590
728 965
713 606
774 977
777 366
802 336
98 705
848 629
263 780
560 -601
759 95
289 -330
874 867
156 265
11 144
415 532
884 584
520 -643
131 -178
493 748
855 100
314 -33
509 -135
14 274
194 937
692 183
693 19
157 -51
626 -768
157 135
789 -911
187 431
703 -52
243 921
728 -158
361 516
757 -405
570 -759
92 464
358 -735
282 -673
495 -883
11 538
469 -530
439 300
784 -845
869 -278
82 -895
96 -37
85 500
461 452
945 189
821 454
75
299 -591
331 -267
385 934
353 -46
8 -644
415 -761
809 893
701 664
440 -591
380 880
710 -160
207 -227
658 -320
848 486
455 -490
422 -369
586 690
188 735
857 -959
491 -931
927 559
204 -693
299 -935
667 41
70 592
287 494
970 -106
512 -395
763 443
388 -879
659 254
130 -794
949 593
544 -570
594 -689
253 -613
513 -914
397 942
431 -114
748 780
548 865
457 552
318 111
854 -302
110 -551
680 -753
119 684
482 -238
333 409
345 650
301 -194
248 -802
707 363
924 -624
26 903
72 258
670 -237
504 -352
713 968
480 118
936 -954
301 -149
217 -50
351 -688
882 -283
39 334
195 -735
284 927
348 -429
297 -791
925 835
884 -466
111 377
76 395
968 643
92
294 -909
219 -113
61 903
309 600
908 418
976 907
538 99
993 38
621 -951
762 -193
766 817
162 -958
352 233
959 -930
243 -702
200 -733
927 186
377 -177
325 406
647 -914
650 501
624 204
817 -706
114 748
142 787
178 913
798 -373
132 -99
965 -954
158 -251
691 370
317 -155
980 533
929 941
367 -957
881 87
325 -550
103 300
747 401
386 -865
589 6
560 -902
16 775
502 318
344 81
237 837
995 59
968 119
761 670
63 245
945 438
45 688
477 -216
951 -866
967 474
794 604
329 -573
552 172
620 -301
446 972
543 63
192 461
330 -835
963 -781
117 325
85 177
94 210
973 934
981 -490
366 -85
249 -62
219 144
311 -595
440 561
912 267
468 -7
863 355
970 20
900 988
733 609
614 78
739 -807
340 -849
374 982
39 -933
363 -1
227 678
621 54
600 155
517 766
413 -379
91 -346
35
353 114
776 -907
81 33
823 -772
500 320
82 182
584 987
799 11
663 -104
895 -457
552 -574
2 -836
910 480
482 597
32 965
949 738
251 -345
166 -755
567 -857
567 -939
340 317
37 -392
903 95
813 -992
433 -861
93 -897
299 283
987 -385
122 -350
925 843
800 135
522 -561
845 -827
175 77
414 -567
33
403 631
399 -305
839 -424
846 -81
118 468
329 627
401 -194
335 -20
642 158
822 -720
664 -16
683 -588
953 -797
302 -77
151 949
62 -840
622 -527
319 -637
714 -230
808 -320
185 299
599 -838
805 140
159 -195
420 -770
485 86
658 -243
318 -415
351 413
499 920
405 -570
219 821
380 -183
