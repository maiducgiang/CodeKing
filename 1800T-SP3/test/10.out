7
9
15
4
12
8
10
10
5
5
